hooks/declarative-subsequent-scans/README.md

```diff
@@ -0,0 +1,116 @@
+---
+title: "Cascading Scans"
+path: "hooks/declarative-subsequent-scans"
+category: "hook"
+type: "processing"
+state: "released"
+usecase: "Enables cascading Scans based declarative _CascadingRules_."
+---
+
+<!-- end -->
+
+## Deployment
+
+Installing the Cascading Scans hook will add a ReadOnly Hook to your namespace which looks for matching _CascadingRules_ in the namespace and start the according scans. 
+
+```bash
+helm upgrade --install dssh ./hooks/declarative-subsequent-scans/
+```
+
+### Verification
+```bash
+$ kubectl get ScanCompletionHooks
```

Comment:
I would recommend removing the **$** at the beginning for more consistency and to make copy and paste more easy